rdr_service/dao/genomics_dao.py

```diff
@@ -3992,7 +4018,9 @@ def get_aw3_wgs_records(self, **kwargs):
                     GenomicSetMember.blockResearch == 1,
                     sqlalchemy.sql.expression.literal("True"),
                     sqlalchemy.sql.expression.literal("False")),
-                GenomicSetMember.blockResearchReason
+                GenomicSetMember.blockResearchReason,
+                GenomicGCValidationMetrics.pipelineId,
+                GenomicGCValidationMetrics.processingCount
```

Comment:
Should we have processingCount + 1 here? I.e. if it is 0 we want to send 1.

Reply:
Should we default it to 1 then in the model?

Reply:
"Proposed new field, will start with 0 the first time an AW3 (WGS) is created and increment by 1 each time a new one is created."

Reply:
If we default to 0 and send it as 1, then we update it below to actually be 1. When we get AW4, it matches Broad. If we default to 1, we send as 1, but when we update it, it is 2, which is out of sync with Broad. I was thinking 0 means we haven't sent it, 1 means first time sending.

Reply:
Understood, made the change